site/js/MessagesActor.js

```diff
@@ -491,42 +491,42 @@ export default class MessagesActor extends Actor {
       else if (char === '_' && currentToken === 'text') {
         if (charNext === '_') {
           // underline
-          toggleStyle('u')
+          await toggleStyle('u')
           c++ // skip charNext
         } else {
           // italic
-          toggleStyle('i_')
+          await toggleStyle('i_')
         }
 
         continue
       }
 
       else if (char === '~' && charNext === '~' && currentToken === 'text') {
         // strikethrough
-        toggleStyle('s')
+        await toggleStyle('s')
         c++ // skip charNext
 
         continue
       }
 
-      else if (char === '$' && charNext === '$' && currentToken !== 'latex') { startToken('latex'); c++; continue }
-      else if (char === '$' && charNext === '$' && currentToken === 'latex') { startToken('text'); c++; continue }
+      else if (char === '$' && charNext === '$' && currentToken !== 'latex') { await startToken('latex'); c++; continue }
+      else if (char === '$' && charNext === '$' && currentToken === 'latex') { await startToken('text'); c++; continue }
 
-      else if (char === '`' && currentToken !== 'code') { startToken('code'); continue }
-      else if (char === '`' && currentToken === 'code') { startToken('text'); continue }
+      else if (char === '`' && currentToken !== 'code') { await startToken('code'); continue }
+      else if (char === '`' && currentToken === 'code') { await startToken('text'); continue }
```

Comment:
JS should just remove `await` and add a new `no-await` keyword, and also make all functions async by defualt :^)

Reply:
yes